File: src/main/java/thkoeln/dungeon/player/domain/PlayerRepository.java
package thkoeln.dungeon.player.domain;

import org.springframework.data.repository.CrudRepository;
import thkoeln.dungeon.game.domain.Game;

import java.util.List;
import java.util.UUID;

public interface PlayerRepository extends CrudRepository<Player, UUID> {
    List<Player> findAll();
    List<Player> findByGameParticipations_Game( Game game );
}

File: src/main/java/thkoeln/dungeon/player/domain/PlayerRepository.kt
package thkoeln.dungeon.player.domain

import org.springframework.data.repository.CrudRepository
import thkoeln.dungeon.game.domain.Game
import java.util.*

interface PlayerRepository : CrudRepository<Player?, UUID?> {
    override fun findAll(): MutableList<Player?>
    fun findByGameParticipations_Game(game: Game?): List<Player?>?
    abstract fun save(player: Player?)
}